### PR TITLE
GDB-11133 - Improve test stability in CI

### PR DESCRIPTION
**Pull request by DesiBorisova**
## What
This improvement attempts to fix some of the test flake in a CI environment.

## Why
Flaky tests cause builds to fail too frequently.

## How
I added checks to verify the view has fully loaded and responses handled before the test starts.

## Testing
Test improvements

## Screenshots
N/A

## Checklist
- [x] Branch name
- [x] Target branch
- [x] Commit messages
- [x] Squash commits
- [x] MR name
- [x] MR Description
- [x] Tests

### CODE REVIEW

**Comment by DesiBorisova** [comment]
Jenkins retry build

File: test-cypress/integration/home/rdf-resource-search.spec.js [hunk 1]
```diff
@@ -3,9 +3,18 @@ import HomeSteps from '../../steps/home-steps';
 const FILE_TO_IMPORT = 'wine.rdf';
 
 describe('RDF resource search', () => {
-
+    let repositoryId;
     beforeEach(() => {
         cy.viewport(1280, 1000);
+        repositoryId = '23repo' + Date.now();
```
Comment (svilenvelikov) [inline test-cypress/integration/home/rdf-resource-search.spec.js:9]: What is `23repo`?

Comment (svilenvelikov) [reply test-cypress/integration/home/rdf-resource-search.spec.js:9]: ahh I now saw that you've just copy-pasted it

File: test-cypress/steps/visual-graph-steps.js [hunk 2]
```diff
@@ -63,9 +67,18 @@ export class VisualGraphSteps {
...
+        this.getAutocompleteToast().should('not.exist');
         cy.get('.result-item.active').should('contain', fullName).click();
     }
 
+    static getAutocompleteToast() {
+        return cy.get('.autocomplete-toast a');
+    }
+
+    static spyAutocompleteStatus() {
+        cy.intercept('GET', '/rest/autocomplete/enabled').as('autocompleteStatus');
```
Comment (svilenvelikov) [inline test-cypress/steps/visual-graph-steps.js:79]: Request interceptors should be in stubs

Comment (DesiBorisova) [reply test-cypress/steps/visual-graph-steps.js]: I moved it to a stubs file

**Review by svilenvelikov — DISMISSED** [review]

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**